$MirOS: ports/www/firesomething/patches/patch-security_nss_lib_freebl_Makefile,v 1.1 2008/03/31 21:33:57 bsiegert Exp $
--- security/nss/lib/freebl/Makefile.orig	2006-12-07 02:59:41.000000000 +0100
+++ security/nss/lib/freebl/Makefile	2008-05-20 21:37:14.000000000 +0200
@@ -192,7 +192,7 @@ endif
 # to bind the blapi function references in FREEBLVector vector
 # (ldvector.c) to the blapi functions defined in the freebl
 # shared libraries.
-ifeq (,$(filter-out BSD_OS FreeBSD Linux NetBSD, $(OS_TARGET)))
+ifeq (,$(filter-out MidnightBSD MirBSD BSD_OS FreeBSD Linux NetBSD, $(OS_TARGET)))
     MKSHLIB += -Wl,-Bsymbolic
 endif
 
